branding_guidelines.md

```diff
@@ -0,0 +1,90 @@
+# 🎨 Branding guidelines
+
+Do you want to use our logos and branding for your use?\
+Here we've written some guidelines for proper use of our logos.
+
+## 📜 Guidelines
+
+| Do this                                                         | Don't do this                                                       |
+| ------------------------------------------------------------ | ------------------------------------------------------------ |
+| :heavy_check_mark: Use our logos to link to Devicon | :x: Use our logos for your own application’s icon |
+| :heavy_check_mark: Use our logos in a blog post or news article about Devicon | :x: Create a modified versions of our logos |
+| :heavy_check_mark: Use the right format for your project to ensure proper resolution | :x: Change the current original provided sizes                  |
```

Comment:
in my opinion, this contradicts 

```
Adapt — remix, transform, and build upon the material for any purpose, even commercially.
```
https://github.com/devicons/devicon/pull/905/files?short_path=48de610#diff-4673a3aba01813b595de187a7a6e9e63a3491d55821606fecd9f13a10c188a1dR8

Reply:
> in my opinion, this contradicts
> 
> ```
> Adapt — remix, transform, and build upon the material for any purpose, even commercially.
> ```
> 
> https://github.com/devicons/devicon/pull/905/files?short_path=48de610#diff-4673a3aba01813b595de187a7a6e9e63a3491d55821606fecd9f13a10c188a1dR8

Should we change to this license, then? https://creativecommons.org/licenses/by-nd/4.0/